Yafc/Properties/launchSettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "profiles": {
+    "Yafc": {
+      "commandName": "Project",
+      "environmentVariables": {
+        " SDL_RENDER_DRIVER": "direct3d12"
+      },
+      "nativeDebugging": false
+    },
+    "installed-release": {
+      "commandName": "Executable",
+      "executablePath": "F:\\yafc\\release\\Windows\\Yafc.exe",
```

Comment:
From what I understand, this line is user-dependent, because my Yafc is not located on the `F` drive.

Reply:
All three added files should have been ignored instead; it's just that our .gitignore file didn't have them. [This](https://github.com/github/gitignore/blob/main/VisualStudio.gitignore) is a common and thorough .gitignore file. (It's used by .NET, among others)

I'd go ahead and add it now, unless @shpaass would prefer to have it separately.

Reply:
Oh, whoops! You know what happened, is that Visual Studio's git plugin automatically ignores them but sometimes I do git from wsl because it's easier to rebase, and that caught them. Will get rid of them if not yet done